release-tools/DO-RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Documentation on the do-release script
+
+The do-release.pl script copies distributions from the temporary holding area
+to the http and ftp areas. It it intended to be run as the `openssl` user on
+dev.openssl.org.
+
+It does the following:
+
+1. Copy OpenSSL release files from the holding area to the http and ftp
+   locations: currently /v/openssl/www/source and /v/openssl/ftp/source
+2. Move OpenSSL release files from holding area to ~openssl/dist/old By
+   doing this the script wont try and make a release again with old files.
+3. Mail the release message. This is sent to openssl-dev openssl-users and
```

Comment:
This is openssl-project now not openssl-dev. As noted below, all it actually does by default is to print the command you need to run - rather than actually run it.

Reply:
Fixed